Source/WebKit/WebProcess/Automation/WebAutomationSessionProxy.cpp

```diff
@@ -527,7 +527,7 @@ void WebAutomationSessionProxy::resolveChildFrameWithNodeHandle(WebCore::PageIde
         return;
     }
 
-    auto* coreFrameFromElement = dynamicDowncast<WebCore::LocalFrame>(downcast<WebCore::HTMLFrameElementBase>(*coreElement).contentFrame());
+    auto* coreFrameFromElement = downcast<WebCore::HTMLFrameElementBase>(*coreElement).contentFrame();
```

Comment:
We could downcast to HTMLFrameOwnerElement here instead (one level up but still has contentFrame.)

I think this is just a nit though.